rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -195,7 +193,6 @@ public void setCoordinatorURI(URI uri) {
             client = ClientBuilder.newClient();
 
         base = uri;
-        target = client.target(base);
```

Comment:
Was this superfluous before?

Reply:
It wasn't used. When I started out I was caching the target but I found resteasy was closing the connection so now I create a new connection on every request. We can revisit the issue when we address performance.